script.js

```diff
@@ -0,0 +1,44 @@
+window.addEventListener('load', function() {
+    /* Write your solution between this comment */
+/*Heather Sleyster
+sleyster_assignment01b
+INFO 2134
+Thoendel
+03/08/2020
+*/
+
+/*These statements assign the select tag to the variable s 
+and the option tags to the o array.
+*/
+let s = document.createElement('select');
+let o = document.getElementsByTagName('option');
+
+/*This for loop populates the o array with numeric values up
+to 100 and appends them to the option tag.
+*/
+for (let i = 0, max = 100; i < max; i ++) {
+s.textContent = o[i];
+s.appendChild('option');
+}
+/*This is the error check event listener to check if all fields
+are filled out
+*/
+document.form.addEventListener('click', errorCheck);

```

Comment:
I think the assignment says to use the form submit event and prevent the default action...